Symfony/CS/Tokenizer/Transformer/ArrayTypehint.php

```diff
@@ -28,8 +28,11 @@ public function process(Tokens $tokens)
     {
         foreach ($tokens->findGivenKind(T_ARRAY) as $index => $token) {
             $nextIndex = $tokens->getNextMeaningfulToken($index);
-            $nextToken = $tokens[$nextIndex];
+            if (null === $nextIndex) {
+                return;
```

Comment:
kind of works, however `'CT_ARRAY_TYPEHINT'` is not in https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.11/Symfony/CS/Tokenizer/Token.php#L247 so the psr0 test fails. Should the custom token not be in that list?